database/seeder/backup/wip.go

```
@@ -0,0 +1,180 @@
+			Slug: fmt.Sprintf("%s-slug", name),
+		}
+		db.Create(&c)
+		cats = append(cats, c)
+	}
+	return cats
+}
+
+func linkCategoriesToPosts(db *gorm.DB, posts []database.Post, cats []database.Category, rng *rand.Rand) {
+	for _, post := range posts {
+		n := rng.Intn(1)
+		db.Model(&post).Association("Categories").Replace(randomSubset(cats, n, rng))
```

golangci-lint (1.64.8)

Error return value of `(*gorm.io/gorm.Association).Replace` is not checked

(errcheck)